<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TubeUiItem</class>
 <widget class="QWidget" name="TubeUiItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TubeUiItem[state='1'] {
	background-color: rgb(224, 255, 238);
}

#TubeUiItem[state='0'] {
	background-color:rgb(255, 255, 255);
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="87,45">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_image" native="true">
         <property name="styleSheet">
          <string notr="true">image: url(:/suzhou/tube_incomplete.png);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_bottom">
       <property name="styleSheet">
        <string notr="true">width: 12px;
height: 18px;
font-family: PingFang SC;
font-weight: 500;
font-size: 23px;
color: #333333;
line-height: 34px;</string>
       </property>
       <property name="text">
        <string>botoom</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_right">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 37px;
height: 19px;
font-family: PingFang SC;
font-weight: 500;
font-size: 20px;
color: #333333;
line-height: 42px;</string>
     </property>
     <property name="text">
      <string>right</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
